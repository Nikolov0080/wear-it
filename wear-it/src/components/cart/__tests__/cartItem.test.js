import React from 'react';
import renderer from 'react-test-renderer';
import CartItem from '../cartItem';

it('renders correctly Cart Item', () => {
    const tree = renderer
        .create(<CartItem
            token={'askdjasdlkdjkj4k23l4j3k2lrkwejrk234jkr2lew'}
            item={
                {
                    productName: 'name',
                    price: '40', size: "XL",
                    imageURL: 'testImageUrl',
                    _id: 'testID'
                }}
        />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});