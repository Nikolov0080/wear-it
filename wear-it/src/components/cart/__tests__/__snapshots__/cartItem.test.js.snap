// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Cart Item 1`] = `
<div
  className="box"
>
  <img
    alt="clothes"
    className="image"
    src="testImageUrl"
  />
  <div
    className="details"
  >
    <div
      className="info"
    >
      Name: 
      name
    </div>
    <div
      className="info"
    >
      Price: 
      40
       USD
    </div>
    <div
      className="info"
    >
      Size: 
      XL
    </div>
  </div>
  <div
    className="btnGroup"
  >
    <div
      className="orderBtn"
      onClick={[Function]}
    >
      Order
    </div>
    <div
      className="deleteBtn"
      onClick={[Function]}
    >
      Remove
    </div>
  </div>
</div>
`;
